packages/material/scss/_variables.scss

```
@@ -1966,7 +1966,7 @@ $upload-focused-shadow: 0 0 0 2px rgba(0, 0, 0, .13) !default;
 
 
 // Breadcrumb
-$breadcrumb-padding-x: $padding-x-sm !default;
+$breadcrumb-spacer-x: $padding-x !default;
```

```suggestion
$breadcrumb-padding-x: $breadcrumb-padding-x !default;
```

Also, add `$breadcrumb-padding-y` variable